### ISSUE TITLE
dnt exiting normally even though tests fail

### ISSUE THREAD

**Issue by sgwilym** [denoland/dnt#86]
I'm setting up some Github workflows that include `dnt`, and am testing against Node v12, v14, and v16.

I expect my build script to fail with v12 and v14 at this point, but v14 is succeeding even though a problem is encountered during tests: https://github.com/earthstar-project/stone-soup/runs/4738270806?check_suite_focus=true#step:5:135

**Comment by chronark** [comment]
I'm also running into this

the test fails on node14 as expected but dnt still finishes with exitCode 0
```

(node:11011) UnhandledPromiseRejectionWarning: ReferenceError: EventTarget is not defined
    at Object.<anonymous> (/Users/andreas/github/chronark/upstash-kafka/npm/node_modules/@deno/shim-deno/dist/deno/stable/classes/PermissionStatus.js:7:32)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at Object.<anonymous> (/Users/andreas/github/chronark/upstash-kafka/npm/node_modules/@deno/shim-deno/dist/deno/stable/classes/Permissions.js:5:31)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:11011) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:11011) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
[dnt] Complete!
```

**Comment by chronark** [comment]
I submitted a fix #89

**Comment by cinnamon-bun** [comment]
Related: https://github.com/denoland/node_deno_shims/pull/80 in shim-deno.

It's fixed there, we just need to get an npm release.
